src/components/order/ReceiverModal.tsx

```diff
@@ -0,0 +1,72 @@
+import { css } from "@emotion/react";
+import OrderForm from "@/components/order/OrderForm";
+import type { FormData } from "@/components/order/OrderForm";
+import type { Theme } from "@emotion/react";
+
+const ReceiverModal = ({
+  onClose,
+  onSave,
+  savedReceiverInfo,
+}: {
+  onClose: () => void;
+  onSave: (formData: FormData) => void;
+  savedReceiverInfo: {
+    receiverName: string;
+    phoneNumber: string;
+    quantity: number;
+  }[];
+}) => {
+  return (
+    <>
+      <div css={OverlayStyle} />
+      <div css={ModalStyle}>
+        <h2>받는 사람</h2>
+        <p css={InfoTextStyle}>* 최대 10명까지 추가 할 수 있어요.</p>
+        <p css={InfoTextStyle}>
+          * 받는 사람의 전화번호를 중복으로 입력할 수 없어요.
+        </p>
+        <OrderForm
+          onSubmitCallback={(formData) => {
```

Comment:
```suggestion
          onSubmitCallback={onSave}
```

이거랑 같은 거 같은데 결과가 달라지는지 한 번 확인부탁드려용!